bundle/manifests/oadp-operator.clusterserviceversion.yaml

```diff
@@ -831,19 +831,19 @@ spec:
   provider:
     name: Red Hat
   relatedImages:
-    - image: quay.io/konveyor/velero:latest
+    - image: quay.io/konveyor/velero:konveyor-1.9
```

Comment:
@sseago do we want change the image tags from `latest` in master ?

Reply:
@shubham-pampattiwar Yes -- for 2 reasons:
1. konveyor-dev is currently at the point just before 1.9 was branched, but there's a bug with downloadrequests there that was fixed on 1.9 (and probably also on main)
2. We can't use current main since there are not-for-1.9 CRD changes there already

Once oadp-1.1 branches are created in a month or so we'll re-point oadp master to latest.

Reply:
The downloadrequest issue is why ci tests were failing with the earlier version of this PR pointing to latest.